packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -893,6 +952,31 @@ const QueryBuilderGridResult = observer(
               onColumnPivotChanged={onSaveGridColumnState}
               onColumnPivotModeChanged={onSaveGridColumnState}
             />
+          ) : isEnterpriseVersionEnabled ? (
```

Comment:
lets try and clean this up a bit, we have too many flags going. 
Think about maybe using enum for modes, 
or maybe even using the flag at the top level when we call <QueryBuilderGridResult>
and using two separate components